core/src/ldml/ldml_processor.cpp

```diff
@@ -168,21 +222,83 @@ ldml_processor::process_event(
 
     switch (vk) {
     case KM_KBP_VKEY_BKSP:
-      state->context().pop_back();
-      state->actions().push_backspace(KM_KBP_BT_UNKNOWN); // Assuming we don't know the character
+      {
+        KMX_DWORD last_char = 0UL; // Assuming we don't know the character
```

Comment:
IMO the comment in this line doesn't make sense - we're trying to get the last char, so of course we don't know it here.

Reply:
```suggestion
        KMX_DWORD last_char = 0UL;
```

was an old comment